charts/amazon-cloudwatch-observability/templates/operator-clusterrole.yaml

```diff
@@ -5,19 +5,31 @@ metadata:
 rules:
 - apiGroups: [ "" ]
   resources: [ "configmaps" ]
-  verbs: [ "create", "delete", "get", "list", "patch", "update", "watch" ]
+  verbs: [ "create","get","list", "watch" ]
```

Comment:
Feels odd to say we the Operator can create something but it cannot patch it or delete it after unless allowlisted in the list below.
Any particular reason to have `create` in this list and not in the restricted one below?

Reply:
create isn't supported in `resourceNames`.

According to https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources

```
Note:
You cannot restrict create or deletecollection requests by their resource name. For create, this limitation is because the name of the new object may not be known at authorization time. If you restrict list or watch by resourceName, clients must include a metadata.name field selector in their list or watch request that matches the specified resourceName in order to be authorized. For example, kubectl get configmaps --field-selector=metadata.name=my-configmap
```